src/controllers/controller.doughnut.js

```diff
@@ -137,6 +135,12 @@ export default class DoughnutController extends DatasetController {
 		this.outerRadius = undefined;
 		this.offsetX = undefined;
 		this.offsetY = undefined;
+		let hidden = hiddenIndices.get(chart);
+		if (!hidden) {
+			hidden = {};
+			hiddenIndices.set(chart, hidden);
```

Comment:
I think this will memory leak the chart since its never removed. Happy to keep this on the `chart` instance if that makes it simpler.

Reply:
True, reverted to storing in chart, and added getter to make it fully usable through API.